### DISCUSSION TITLE
개발자님 질문이여

### DISCUSSION THREAD

**Discussion by YunYeongbin** [changdoc/naver-vaccine-macro#250] [Q&A]
매크로 돌리는 중인데

Access to fetch at 'https://api.place.naver.com/graphql' from origin 'https://m.place.naver.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

콘솔창에 이렇게 떠도 상관이 없는건가요?

**Comment by changdoc** [comment]
아뇨 문제가 있는 상황 같네요 페이지 새로 고침 후에 동작 해도 동일한 에러가 계속 발생하고 있을까요?

**Reply by YunYeongbin** [reply]
넵 처음 몇 초는 그냥 선택 백신 종류: 화이자, ~~ 이렇게 뜨는데
시간이 좀 지나면 계속 저렇게 뜨네요

**Reply by changdoc** [reply]
동일한 증상을 겪는 분들이 몇 분 계셨는데 해결방법을 못 찾은 상황입니다.
카카오 맵에서 진행하시거나 다른 매크로를 이용하셔야할 것 같습니다.
